.github/PULL_REQUEST_TEMPLATE/pull-request-template.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Pull Request
+about: Pull Request Template
+title: ""
+labels: Please review - In progress - Easy - Medium - Hard - Do not merge - Change log - Release notes - Many requirements - 
+assignees: ""
+---
+
+Thank you for your Pull Request!
+
+Before you submit, please check that:
+
+- [ ] You mentioned a related issue if this PR related to it, e.g. `Fixes #228` or `Related #1337`.
+- [ ] You added a brief description of the PR, e.g.:
+  - What does it do?
+  - What important points reviewers should know?
+
+- [ ] You labeled the PR appropriately with the labels listed in the sidebar
+- [ ] PR status, changelog, release notes, various implications/requirements
+- [ ] You asked any particular reviewers to review. If you aren't sure, start with GH suggestions
+- [ ] Your PR adheres to the style guide
+  - In particular, mind the maximal line length of 100 (120 in exceptional circumstances)
```

Comment:
Does this PR adhere to the line length standard, @krichard410? :wink:

Reply:
Ah good point there! I'm going to take this line out since I believe most people don't follow this.

Reply:
I would absolutely recommend enforcing a line length rule specifically because people will not pay attention to this unless you force them to and unfortunately I kind of think common tools like GitHub make it important.